pkg/controllers/multiclusterservice/endpointslice_collect_controller.go

```diff
@@ -146,7 +146,7 @@ func (c *EndpointSliceCollectController) collectEndpointSlice(key util.QueueKey)
 	return nil
 }
 
-func (c *EndpointSliceCollectController) buildResourceInformers(ctx context.Context, work *workv1alpha1.Work, clusterName string) error {
+func (c *EndpointSliceCollectController) buildResourceInformers(_ context.Context, _ *workv1alpha1.Work, clusterName string) error {
```

Comment:
```suggestion
func (c *EndpointSliceCollectController) buildResourceInformers(clusterName string) error {
```
I think we can remove the two parameters. 
cc @jwcesign to confirm.

Reply:
+1